cypress/support/fragments/orders/orders.js

```diff
@@ -0,0 +1,50 @@
+import { Button, SearchField, PaneHeader, Pane } from '../../../../interactors';
+
+const actionsButtonName = 'Actions';
+
+export default {
+
+  createOrderWithOrderLineViaApi(order, orderLine) {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
may be excess getToken. My current expectation that it's enough to execute getToken at the beginning of the test and after this send any requests with specified token from sys env variables.

Reply:
Updated